README.md

```diff
@@ -42,6 +42,18 @@ and send this message object to the aggregator using the WebSocket connection.
 
 The tests for the Solid Stream Aggregator are written using the Jest framework. The coverage isn't 100% yet, but will be done in the recent future.
 
+## Linting
+
+You run the linter via 
+```shell
+npm run lint:ts
```

Comment:
running this gives me an output as,
```bash

✖ 535 problems (101 errors, 434 warnings)
  0 errors and 2 warnings potentially fixable with the `--fix` option.
  ```
  So, the linting looks good to me. I will try to fix theese errors and warnings. Would add tests after this in a seperate pull request.
  
  @pheyvaer Should I ping you once,
  1. The linting errors are sorted.
  2. Most of the coverage is done.

Reply:
Ok in that case

1) Merge this PR.
2) Create a new branch were you fix the linting errors. The corresponding issue is https://github.com/SolidLabResearch/solid-stream-aggregator/issues/30
3) Make a PR with that new branch.
4) Assign me as a reviewer.